sdk/src/contexts/solana/context.ts

```diff
@@ -404,7 +404,7 @@ export class SolanaContext<
     );
 
     transaction.feePayer = payerPublicKey;
-    await addComputeBudget(this.connection!, transaction, [NATIVE_MINT]);
```

Comment:
We don't need to pass the accounts anymore - `addComputeBudget` finds them for us